pkg/skaffold/deploy/kpt/kpt.go

```diff
@@ -125,11 +118,17 @@ func versionCheck(dir string, stdout io.Writer) error {
 		match := re.FindStringSubmatch(versionInfo)
 		if len(match) != 2 {
 			color.Yellow.Fprintf(stdout, "unknown kustomize version %q\n"+
-				"Your kustomize may be not from the official release\n%v\n", string(out),
-				kustomizeFurtherGuidance)
+				"Your kustomize may be not from the official release\n"+
+				"Please make sure your local kustomize version >= the official version %v, "+
+				"otherwise some features may not be well supported. "+
```

Comment:
Can we shorten and simplify this to:
```suggestion
				"Skaffold/Kpt requires kustomize %v or later.\n"+
```

Reply:
skaffold/kpt does not *require* the kustomize to be >= a certain version. It is just that if users encounter some issues, we remind them that the old kustomize version may be the cause. This warning is especially helpful here because there's no kpt/kustomize version skew support yet.

Reply:
That sounds like there’s a minimum version! Would _recommended_ work?

(I don’t think we’re doing anybody a favour if they’re using an older version that we suspect may lead to problems.)

Reply:
good idea. 😄 Updated.